src/main/java/core/basesyntax/Factorial.java

```diff
@@ -6,6 +6,10 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        int result = 1;
+        for (int i = 1; i < number + 1; i++) {
+            result = result * i;
+        }
+        return result;
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_06.html check checklist about requirements  to this task